spk/shairport-sync/Makefile

```diff
@@ -34,3 +34,4 @@ include ../../mk/spksrc.spk.mk
 shairport_extra_install:
 	install -m 755 -d $(STAGING_DIR)/var
 	install -m 644 src/alsa.conf $(STAGING_DIR)/share/alsa/alsa.conf
+	install -m 600 src/shairport-sync.conf $(STAGING_DIR)/var/shairpot-sync.conf
```

Comment:
Please set this to 644 (which is default...for some reason, Couch doesn't adhere to that unfortunately, we'll need to fix that)